plugins/module_utils/core.py

```diff
@@ -55,7 +55,7 @@
     json_patch,
     get_patch,
 )
-
+from .utils import ansible_dict_to_boto3_tag_list # pylint: disable=unused-import
```

Comment:
Please add also 
`from .utils import snake_dict_to_camel_dict # pylint: disable=unused-import`

And remove those imports from line 46.